packages/babel-preset-react-app/index.js

```diff
@@ -76,6 +76,9 @@ module.exports = function(api, opts) {
           // Adds component stack to warning messages
           // Adds __self attribute to JSX which React will use for some warnings
           development: isEnvDevelopment || isEnvTest,
+          // Will use the native built-in instead of trying to polyfill
```

Comment:
The changes in the babel config, how do they relate to the workerbox? 

Should it be done in seperate PR?

Reply:
Sorry, at the time, next was broken and I had to merge #4159 as it was at the time. I'll rebase